pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.de-de.md

```diff
@@ -0,0 +1,301 @@
+This way, all Public Cloud instances hosted on it can directly access the MongoDB service.
+
+```bash
+resource "ovh_cloud_project_database_mongodb_user" "userjd" {
+resource "ovh_cloud_project_database_ip_restriction" "iprestriction" {
+  service_name = ovh_cloud_project_database.service.service_name
+  engine       = ovh_cloud_project_database.service.engine
+  cluster_id   = ovh_cloud_project_database.service.id
+  ip           = "192.168.12.0/24"
+}
+```
+
+Of course, you can be more restrictive and provide several IP addresses separately in /32, to target only certain authorised sources.
+
+#### Step 5 - User management
+
+To access the MongoDB service, you need to be an authenticated user.
+By default, the cluster will be delivered with the user `admin@admin`, whose password you can reset via the OVHcloud Control Panel to log in directly.
+
+You can also add another user via Terraform, with full wide read/write permissions on all databases, as follows:
+
+```bash
+resource "ovh_cloud_project_database_mongodb_user" "tf_user" {
+  service_name = ovh_cloud_project_database.service.service_name
+  cluster_id   = ovh_cloud_project_database.service.id
+  name         = "tf_mongodb_user"
+  roles        = ["readWriteAnyDatabase@admin"]
+}
+```
+
+#### Step 6 - Useful information
+
+At the end of the Terraform script, it is always useful to prepare some “outputs” that will provide relevant information to keep in mind at the end of the setup.
+
+In our case:
+
+- the MongoDB connection URI;
+- the login details of the user created by Terraform, including their password.
+
+```bash
+output "cluster_uri" {
+  value = ovh_cloud_project_database.service.endpoints.0.uri
+}
+
+output "user_name" {
+  value = ovh_cloud_project_database_mongodb_user.userjd.name
+}
+
+output "user_password" {
+  value     = ovh_cloud_project_database_mongodb_user.userjd.password
+  sensitive = true
+}
+```
+
+#### Step 7 - Terraform Plan Command
+
+```bash
+$ terraform apply
+[…]
+Plan: 5 to add, 0 to change, 0 to destroy.
+
+Changes to Outputs:
+  + cluster_uri   = (known after apply)
+  + user_name     = "tf_mongodb_user@admin"
+  + user_password = (sensitive value)
+
+Do you want to perform these actions?
+  Terraform will perform the actions described above.
+  Only “yes” will be accepted to approve.
+
+  Enter a value: yes
+
+openstack_networking_network_v2.my_private_network: Creating...
+openstack_networking_subnet_v2.my_subnet: Creating...
+ovh_cloud_project_database.service: Creating...
+[…]
+
+Apply complete! Resources: 5 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+cluster_uri = "mongodb+srv://<username>:<password>@mongodb-xxx-yyy.database.cloud.ovh.net/admin?replicaSet=replicaset&tls=true"
+user_name = "tf_mongodb_user@admin"
+user_password = <sensitive>
+```
+
+### Connection test using an instance of the same private network
```

Comment:
I would add an explanation on  how to access to a private instance through a floating IP with this page the https://help.ovhcloud.com/csm/en-gb-public-cloud-network-attach-floating-ip-to-instance?id=kb_article_view&sysparm_article=KB0050231

Reply:
@yomovh , please let us know if https://github.com/ovh/docs/pull/5730/commits/2b1a7bb090fd78fce8b45cae29c87f75de4445de fits your suggestion